zebra-consensus/src/verify.rs

```diff
@@ -4,7 +4,9 @@
 //! verification.
 
 mod block;
+mod redjubjub;
 mod script;
 mod transaction;
 
+pub use self::redjubjub::{RedJubjubItem, RedJubjubVerifier};
```

Comment:
naming: if the types are called just `Item` and `Verifier`, this reëxport can be dropped and users can choose how much they want to qualify the name, depending on their context, either `redjubjub::Item` and `redjubjub::Verifier` or `Item` and `Verifier`.   I think this is more flexible than including `RedJubjub` in the struct names.

Reply:
(There's then a potential naming overlap between the `redjubjub` crate's `Verifier` type and this one, but this is only ambiguous when code needs to refer to both synchronous and asynchronous batch verification APIs, which only happens in this specific module, otherwise people just use one or the other).

Reply:
I think this would still be good to do.  More rationale: https://github.com/rust-lang/rfcs/pull/356